drivers/dummy/rasterizer_dummy.h

```diff
@@ -0,0 +1,539 @@
+/*************************************************************************/
+/*  rasterizer.h                                                         */
```

Comment:
Should be `rasterizer_dummy.h` (with the ending `*/` properly aligned after the edit).